controller/appcontroller.go

```diff
@@ -1878,7 +1878,7 @@ func (ctrl *ApplicationController) processAppHydrateQueueItem() (processNext boo
 		return processNext
 	}
 
-	ctrl.hydrator.ProcessAppHydrateQueueItem(origApp)
+	ctrl.hydrator.ProcessAppHydrateQueueItem(origApp.DeepCopy())
```

Comment:
Do you remember why the deep copy? Might be worth a comment.

Reply:
We should not modify the object in the informer.  Other queue processors are making deep copy at this level